code/modules/overmap/ships/computers/sensors.dm

```diff
@@ -36,7 +36,7 @@
 		data["range"] = sensors.range
 		data["health"] = sensors.health
 		data["max_health"] = sensors.max_health
-		data["heat"] = sensors.heat
+		data["heat"] = sensors.current_heat
```

Comment:
what's the changes to this file all about?

Reply:
That object was already using the heat war so I renamed it to current_heat instead. It was only used locally by that object.